src/main/java/com/onarandombox/MultiverseCore/utils/MVPlayerLocation.java

```diff
@@ -0,0 +1,159 @@
+/******************************************************************************
+ * Multiverse 2 Copyright (c) the Multiverse Team 2011.                       *
+ * Multiverse 2 is licensed under the BSD License.                            *
+ * For more information please check the README.md file included              *
+ * with this project.                                                         *
+ ******************************************************************************/
+
+package com.onarandombox.MultiverseCore.utils;
+
+import com.dumptruckman.minecraft.util.Logging;
+import com.onarandombox.MultiverseCore.MultiverseCore;
+import com.onarandombox.MultiverseCore.api.MultiverseWorld;
+import com.onarandombox.MultiverseCore.api.MVDestination;
+import com.onarandombox.MultiverseCore.destination.ExactDestination;
+import org.bukkit.Location;
+import org.bukkit.configuration.file.YamlConfiguration;
+import org.bukkit.entity.Player;
+
+import java.io.File;
+import java.util.logging.Level;
+
+public class MVPlayerLocation {
+
+    private static final String PLAYER_LOCATION_DATA = "player_location_data";
+
+    private static MultiverseCore plugin = null;
+
+    public static void init(MultiverseCore p) {
+        plugin = p;
+    }
+
+    private static class InvalidData extends Exception {
+        public InvalidData(String msg) {
+            super(msg);
+        }
+    }
+
+    public static void savePlayerLocation(Player player, Location location, String action) {
+        String world    = location.getWorld().getName();
+        String playerID = player.getUniqueId().toString();
+
+        if (plugin == null)
+            return;
+
+        plugin.log(Level.FINE, "Player '" + player.getName()
+                + "' (" + playerID + ") was in world '" + world + "' at "
+                + Double.toString(location.getX()) + ", "
+                + Double.toString(location.getY()) + ", "
+                + Double.toString(location.getZ()) + ", "
+                + Double.toString(location.getYaw()) + ", "
+                + Double.toString(location.getPitch()) + " before " + action + ".");
+
+        YamlConfiguration yc = new YamlConfiguration();
+        yc.set("schema", 1);
+        yc.set("world", world);
```

Comment:
Don't we have something that handles serializing and deserializing locations for configs?